activity/covid19/covid19_context

```diff
@@ -3,7 +3,6 @@
         "@version": 1.1,
         "wellbeing": "https://raw.githubusercontent.com/ReproNim/reproschema/master/activities/Wellbeing/items/",
```

Comment:
i don't think this is at this location. shouldn't this and the next be changed to the reproschema-library?

Reply:
yes.